magefile.publish.go

```diff
@@ -4,171 +4,129 @@ package main
 
 import (
 	"fmt"
-	"net/url"
 	"os"
-	"path"
+	"os/exec"
 	"path/filepath"
-
-	"github.com/magefile/mage/sh"
 )
 
-const s3UpdateRepoScriptEnv = "S3_UPDATE_REPO_SCRIPT_URL"
-const s3UpdateRepoScriptName = "update_repo.sh"
-const s3BucketEnv = "S3_BUCKET"
-const s3FolderEnv = "S3_FOLDER"
-
 const distPath = "dist"
 
 const packageName = "cartridge-cli"
-const rpmExt = "rpm"
-const debExt = "deb"
 
 type Distro struct {
 	OS   string
 	Dist string
 }
 
 var targetDistros = []Distro{
-	{OS: "el", Dist: "6"},
 	{OS: "el", Dist: "7"},
 	{OS: "el", Dist: "8"},
-	{OS: "fedora", Dist: "29"},
 	{OS: "fedora", Dist: "30"},
+	{OS: "fedora", Dist: "31"},
+	{OS: "fedora", Dist: "32"},
+	{OS: "fedora", Dist: "33"},
+	{OS: "fedora", Dist: "34"},
+	{OS: "fedora", Dist: "35"},
 
-	{OS: "ubuntu", Dist: "trusty"},
 	{OS: "ubuntu", Dist: "xenial"},
 	{OS: "ubuntu", Dist: "bionic"},
 	{OS: "ubuntu", Dist: "eoan"},
 	{OS: "ubuntu", Dist: "focal"},
+	{OS: "ubuntu", Dist: "hirsute"},
 
-	{OS: "debian", Dist: "jessie"},
 	{OS: "debian", Dist: "stretch"},
 	{OS: "debian", Dist: "buster"},
 	{OS: "debian", Dist: "bullseye"},
 }
 
-func getArch(distro Distro) (string, error) {
-	if distro.OS == "el" || distro.OS == "fedora" {
-		return "x86_64", nil
-	}
+func walkMatch(root string, patterns []string) ([]string, error) {
```

Comment:
Please add a function description (with a dot at the end). I don't really have a strong association with what `walkMatch` is supposed to do.

Reply:
Added

Reply:
`Walk` -> `walkMatch`
https://go.dev/blog/godoc
> "Notice this comment is a complete sentence that begins with the name of the element it describes."

Reply:
Fixed